### ISSUE TITLE
Npm install not working

### ISSUE THREAD

**Issue by kris-luminar** [dimavroudis/cy-graphql#17]
I tried installing this via the following methods:
```bash
> npm install @mavrou/cy-graphql@2.0.0
> npm install @mavrou/cy-graphql@1.0.0
> npm install --save-dev @mavrou/cy-graphql
> npm install @mavrou/cy-graphql
```
In each case, the executable code appears to be missing:
```bash
> tree node_modules/@mavrou
node_modules/@mavrou
└── cy-graphql
    ├── LICENSE
    ├── README.md
    └── package.json
```
the package.json file in the `tree` command above shows this line
```json
"files": [
    "/dist/*"
  ],
```
which I think is looking for a folder called `dist` that is not in the `node_modules/@mavrou/` folder.

**Comment by kris-luminar** [comment]
Also, I think you want either `./dist` or just `dist`, not `/dist/` which points to root.

**Comment by dimavroudis** [comment]
Hey @kris-luminar. Thanks for reporting the issue. 

It was an issue with the Github action that released the npm package. I fixed it on version 2.0.1.

**Comment by kris-luminar** [comment]
Thanks so much @dimavroudis !
